content/en/apps/tutorials/tasks-1.md

```diff
@@ -0,0 +1,106 @@
+---
+title: "Configuring A Simple Task"
+linkTitle: Tasks
+weight: 9
+description: >
+  Writing and testing a simple task
+relatedContent: >
+  apps/features/tasks
+  apps/reference/tasks
+  apps/concepts/workflows
+  design/best-practices#anatomy-of-a-task
+
+---
+
+{{% pageinfo %}}
+Tasks prompt users to complete activities on a programmatic schedule. This guide will explain how to write a task which prompts CHW users to complete an _assessment_ [app form]{{< ref "apps/tutorials/app-forms" >}} for new patients within 7 days of registration.
+
+- Creating a straight-forward task
+- Running and testing that task
+{{% /pageinfo %}}
+
+## Prerequisites
+
+* Complete the [App Forms Tutorial]({{< ref "apps/tutorials/app-forms" >}}) - Tasks prompt users to _complete activities_ by opening an app form. The app forms tutorial produces an _assessment_ app form which we will use here. You can also elect to substitute that with any [example app form](https://github.com/medic/cht-core/tree/master/config/default/forms/app).
+* Complete the [Contact and User Management - Part 1 Tutorial]({{< ref "apps/tutorials/contact-and-users-2" >}}) to create a hierarchy of contacts and an offline CHW user. 
+* Read [Understanding the data available in tasks and targets]({{< ref "apps/guides/tasks/task-schema-parameters" >}})
+
+## A simple task
+
+The appearance, behaviour, and schedule of tasks are all controlled through the JavaScript in the `tasks.js` file. Let's start in that file with a simple first task:
+
+```javascript
+module.exports = [{
+  name: 'assessment-after-registration',
+  title: 'First Assessment',
+  icon: 'icon-healthcare',
+  appliesTo: 'contacts',
+  appliesToType: ['patient'],
+  appliesIf: c => user.parent && user.parent.contact_type === 'chw_area' && !c.contact.date_of_death && !c.contact.muted,
+  actions: [{ form: 'assessment' }],
+  events: [{
+    start: 7,
+    days: 7,
+    end: 0,
+  }],
+}];
+```
+
+**What is this code doing?**
```

Comment:
We could shorten this section by linking to the [tasks.js](https://docs.communityhealthtoolkit.org/apps/reference/tasks/#tasksjs) reference and having a shorter summary here. It would mean (hopefully) we'd have to maintain one less document if something is updated in [reference/tasks](https://docs.communityhealthtoolkit.org/apps/reference/tasks/#tasksjs).

Reply:
The [angular tutorials](https://angular.io/tutorial/toh-pt2#displaying-heroes) you linked do this... a line-by-line description of what's going on. I personally find it very helpful. The difference is an "abstract description of what can happen" vs a literal description of what is happening for this exact case.